Use Cases/migrate-normal-table-to-timescale-table.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Migrate normal table to timescale table"
+date: 2025-01-15
+tags:
+  - postgresql
+  - timescale
+  - database
+  - hedge foundation
+description: "How do we migrate normal table to timescale table to optimized data storage"
+authors:
+  - minhth
+---
+As high trading volume the data in 2 tables `user_trades` and `incomes` are bigger so it will make our report is slower every day. With this problem, we consider to move these tables to timescale tables to split into multiple chunks by time so it will optimized our query report
```

Comment:
Can we use a scanner or GPT to check the grammar? For example, the paragraph has some obvious grammar mistakes:

![image](https://github.com/user-attachments/assets/b24f44af-bc41-4ca7-babf-c14453cf3a3f)